index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document Microverse</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header id="header">
+        <img id="header-img" src="images/product-landing-page-logo.png">
+        <nav id="nav-bar">
+            <ul>
+                <li><a class="nav-link" href="#features">Features</a></li>
+                <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+            </ul>  
+        </nav>
+    </header>
+    <form id="form"method="Post" action="https://www.freecodecamp.com/email-submit">
+        <input name = 'emai'type="email" placeholder="Enter Your Email" id="email" required />
+        <input type="submit" id="submit" value="submit">
+    </form>
+
+<section id="features"></section>
+<section id="how-it-works">
+    <iframe id="video" width="420" height="345" src="https://www.youtube.com/embed/tgbNymZ7vqY">
+    </iframe>
+</section>
```

Comment:
- [ ] Please try to be more professional in your comments and add solid reasons why a student should follow your advice, even if it seems like your comment is not clear at all and need more explanation. Kindly be more clear and polite in your comments while performing code review.
   
## Your comment

>  Good job +1 As suggested , kindly make the id to class name we use id for no repeatable so use class as mention below